### PR TITLE
Clean up label set error on bundle not found

### PR DESCRIPTION
**Pull request by kerneltime**
Signed-off-by: Ritesh H Shukla <ritesh@oneconcern.com>

### CODE REVIEW

File: cmd/datamon/cmd/flags.go [hunk 1]
```diff
@@ -472,3 +472,7 @@ func requireFlags(cmd *cobra.Command, flags ...string) {
 		}
 	}
 }
+
+func dumpContext(flags flagsT) string {
```
Comment (ransomw1c) [inline cmd/datamon/cmd/flags.go]: suggestion: either

* scope `dumpContext` locally in `label_set.go`'s `Run` `func`

or

* add `dumpContext` as a method to `cliOptionInputs`

this suggestion is based on, most recently, [commentary](https://github.com/oneconcern/datamon/pull/379#discussion_r373773921) on #379 , specifically

> the new `cliOptionInput` (without the cli prefix...) should bear more responsibilities, and be shed from `cmd`

File: cmd/datamon/cmd/label_set.go [hunk 2]
```diff
@@ -44,7 +44,7 @@ Setting a label is analogous to the git command "git tag {label}".`,
 			return
 		}
 		if !bundleExists {
-			wrapFatalln(fmt.Sprintf("bundle %v not found", bundle), nil)
+			wrapFatalln(fmt.Sprintf("bundle:%s not found in repo:%s %s ", bundle.BundleID, bundle.RepoID, dumpContext(datamonFlags)), nil)
```
Comment (ransomw1c) [inline cmd/datamon/cmd/label_set.go]:
```suggestion
			wrapFatalln(fmt.Sprintf("bundle:%s not found in repo:%s %s ", bundle.BundleID, bundle.RepoID, optionInputs.dumpContext()), nil)
```

File: cmd/datamon/cmd/flags.go [hunk 3]
```diff
@@ -571,3 +571,7 @@ func requireFlags(cmd *cobra.Command, flags ...string) {
...
```
Comment (ransomw1c) [inline cmd/datamon/cmd/flags.go]:
```suggestion
func (in *cliOptionInputs) dumpContext() string {
flags := in.params
```

File: cmd/datamon/cmd/flags.go (hunk 1 above)
Comment (kerneltime) [reply cmd/datamon/cmd/flags.go]: ok

File: cmd/datamon/cmd/flags.go (hunk 3 above)
Comment (kerneltime) [reply cmd/datamon/cmd/flags.go]: take a look

**Pull request merged**